### ISSUE TITLE
Documentation Request: Elaborate on caveats of zsh4humans when using vi bindings

### ISSUE THREAD

**Issue by Semnodime** [romkatv/zsh4humans#298]
According to the README.md](https://github.com/romkatv/zsh4humans/edit/master/README.md)
> Zsh for Humans is not a good choice for users who prefer vi bindings in their shell.

Could you elaborate on the reasoning behind this consclusion / opinion  of yours in the README.md file?
I am thankful for (and impressed by) your work regarding `zsh-bench` and wanted to switch my config to make use of `zsh4humans` but plan on using vi bindings.
Therefore, I am interested in understanding the caveats of `zsh4humans` in this regard.

**Comment by romkatv** [comment]
The reasoning is that the experience isn't great. I don't use vi bindings myself, hence sub-part support for them. I don't know what good support would entail, and I don't want to spend time on it.

**Comment by Semnodime** [comment]
Thanks for the fast reply.
I understand that since you don't use vi bindings they have no support built-in.

To clarify, I was wondering whether you regard `zsh4humans` to have some non-obvious implementation/configuration detail which makes it less suited (e.g. as in incompatible) to configure vi bindings even if one is willing to configure it and does not expect the bindings to work 'out ouf the box'.

**Comment by romkatv** [comment]
There is nothing in zsh4humans that makes it inherently incompatible with vi mode. You just have to define your own bindings.

**Comment by Semnodime** [comment]
I just noticed #230. 

> @dancek There will be bugs when using vi mode
> @romkatv nothing makes it inherently incompatible with vi mode

1.) I wonder whether functional vi bindings can be achieved by extending .zshrc while not modifying zsh4humans itself.

> @dancek doesn't imply any kind of support
> @romkatv sub-part support, I don't want to spend time on it, just define your own bindings

2.) The problem with wording it as 'z4h does not support' is that depending on the interpretation of 'support' it might either imply "maintainer won't spend time on it. don't ask" or it might imply "won't ever work/can't be made to work", with the latter being technically false (AFAIK).

I personally value accurate wording over "protecting against" users who request support or submit issues. I, however, am not in the role of maintaining z4h, so I don't know if those values are misaligned regarding the burden of maintenance.

TLDR I just wanted to know if making functional vi bindings is possible while using z4h since I regard z4h as a performant project probably suited for my needs. Yet, I wanted to avoid switching to z4h if others noticed deal-breaking incompatibilities. I thought consulting the README.md was best suited to gather this information to avoid having to bug the maintainer unnecessarily. Somehow this didn't work out and I am still somewhat clueless regarding my initial question.

**Comment by romkatv** [comment]
I see your point. I can confirm that there is nothing in z4h that makes it impossible to configure vi bindings in `.zshrc` without modifying z4h. However, I don't want to spend my time on supporting (as in actively doing something related to) vi mode, including documentation changes. This applies generally to features I don't personally use. What I can promise is that I won't break your shell with future updates even if you use vi mode.

Does this help clarify the current state of affairs?
